src/realm/util/interprocess_condvar.cpp

```diff
@@ -190,8 +241,10 @@ void InterprocessCondVar::init_shared_part(SharedPart& shared_part)
 
 void InterprocessCondVar::release_shared_part()
 {
-#ifdef REALM_CONDVAR_EMULATION
+#ifdef REALM_CONDVAR_EMULATION 
+#ifndef _WIN32
     File::try_remove(m_resource_path);
+#endif
 #else
 #endif
```

Comment:
I actually prefer it like this. The reason is that if we need to support other platforms, the empty "else" case indicates to you that you might need to fill out that case. It has the same ifdef-structure as in the other methods that happen to have actual contents there